<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smartCalc</class>
 <widget class="QMainWindow" name="smartCalc">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>620</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">smartCalc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/prefix1/calculator.png</normaloff>:/new/prefix1/calculator.png</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F7F8FB;</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_action_deleteAllChar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #858585;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>Ac</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_deleteLastChar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #858585;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_toShare">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #ADE2FF;
	color: #109DFF;
	border-radius: 16px;
	font-size: 32px;
	font-weight: 600;
}

QPushButton::hover{
	background: #85D3FF;
}

QPushButton:pressed{
	background: #72CCFF;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_multiply">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #ADE2FF;
	color: #109DFF;
	border-radius: 16px;
	font-size: 32px;
	font-weight: 600;
}

QPushButton::hover{
	background: #85D3FF;
}

QPushButton:pressed{
	background: #72CCFF;
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_minus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #ADE2FF;
	color: #109DFF;
	border-radius: 16px;
	font-size: 32px;
	font-weight: 600;
}

QPushButton::hover{
	background: #85D3FF;
}

QPushButton:pressed{
	background: #72CCFF;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_plus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>62</width>
      <height>96</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #ADE2FF;
	color: #109DFF;
	border-radius: 16px;
	font-size: 32px;
	font-weight: 600;
}

QPushButton::hover{
	background: #85D3FF;
}

QPushButton:pressed{
	background: #72CCFF;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_0">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>510</y>
      <width>131</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_equally">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>472</y>
      <width>62</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #109DFF;
color: #ADE2FF;
border-radius: 16px;
font-size: 32px;
font-weight: 600;
}

QPushButton::hover{
	background: #0091F5;
}

QPushButton:pressed{
	background: #0083DD;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit{
background: #ffffff;
border: 2px solid #F7F8FB;
color: #424242;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
padding-left: 15px;
padding-right: 15px;
qproperty-alignment: 'AlignVCenter | AlignRight';
}


QLineEdit::hover{
	background: #FCFCFE;
}

QLineEdit:pressed{
	background: #EFF7FF;
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_bracketLeft">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_bracketRight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_log">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_ln">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_sin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_asin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_acos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_atan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_tan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_cos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_sqrt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_action_up">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>159</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_x">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_additional_x">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>159</y>
      <width>131</width>
      <height>62</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit{
background: #ffffff;
border: 2px solid #F7F8FB;
color: #424242;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
padding-left: 15px;
padding-right: 15px;
}


QLineEdit::hover{
	background: #FCFCFE;
}

QLineEdit:pressed{
	background: #EFF7FF;
}


</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_pi">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>62</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #ffffff;
color: #7CC9FF;
border-radius: 16px;
font-size: 18px;
font-weight: 600;
}

QPushButton::hover{
	background: #FCFCFC;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #FCFCFE;
color: #38B9FF;
border:2px solid #ffffff;
border-radius: 16px;
font-size: 26px;
font-weight: 600;
}


QPushButton::hover{
	background: #ffffff;
}

QPushButton:pressed{
	background: #EFF7FF;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_close_program"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_open_baseCalc"/>
    <addaction name="action_open_creditCalc"/>
    <addaction name="action_open_depositCalc"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAdditional">
    <property name="title">
     <string>Additional</string>
    </property>
    <addaction name="action_open_graphCalc"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuView"/>
   <addaction name="menuAdditional"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_close_program">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_open_baseCalc">
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="action_open_creditCalc">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="action_open_depositCalc">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
  <action name="action_open_graphCalc">
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
